### PR TITLE
feat: add new dataset to the ontological database

### PR DESCRIPTION
**Pull request by CedrickArmel**
Add several new dataset instances to the rhizome dataverse example.

### CODE REVIEW

File: example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    core:hasLience license:LO-FR-1_0 ;
+    core:hasPublisher "OKP4"^^xsd:string ;
+    core:hasSpatialCoverage area:250 ;
+    core:hasTag "alimentation-humaine"^^xsd:string,
+        "aliments"^^xsd:string,
+        "composition-des-aliments"^^xsd:string ;
+    core:hasTemporalCoverage [ a core:Period ;
+            core:hasEndDate "None"^^xsd:dateTime ;
+            core:hasStartDate "None"^^xsd:dateTime ] ;
+    core:hasTitle "Table de composition nutritionnelle des aliments Ciqual 2020"^^xsd:string ;
```
Comment (ccamel) [inline example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl:28]: Missing `@fr`.

File: example/rhizome/dataset/10ca8ff2-f0c3-47ba-b37a-8560b2e41ae7.ttl [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    core:hasDescription "L’OCS GE est une base de données de référence pour la description de l’occupation du sol de l’ensemble du territoire métropolitain et des départements et régions d’outre-mer (DROM). Elle est produite à partir de données existantes extraites des bases de données de l’IGN, et de toutes autres données mobilisables issues de référentiels nationaux ou locaux. Plus récemment, dans le cadre du dispositif de l’artificialisation des sols, elle est également produite grâce à des procédés inédits d’intelligence artificielle."@fr ;
+    core:hasFormat mediatype:application_vndshp ;
+    core:hasImage <https://geoservices.ign.fr/sites/default/files/styles/echelle_recadrage_00/public/2021-06/ocsge_Vignette.png?itok=1hXTpDGh> ;
+    core:hasLience license:LO-FR-1_0 ;
+    core:hasPublisher "OKP4"^^xsd:string ;
+    core:hasSpatialCoverage area:250 ;
+    core:hasTag "occupationocsocsgeocsol"^^xsd:string ;
+    core:hasTemporalCoverage [ a core:Period ;
+            core:hasEndDate "None"^^xsd:dateTime ;
+            core:hasStartDate "None"^^xsd:dateTime ] ;
```
Comment (ccamel) [inline example/rhizome/dataset/10ca8ff2-f0c3-47ba-b37a-8560b2e41ae7.ttl:25]: Not correct: `None` is not an `xsd:dateTime`. If there's no temporal coverage, I'd remove the property `core:hasTemporalCoverage`.

File: example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+@prefix mediatype: <https://ontology.okp4.space/thesaurus/media-type/> .
+@prefix meta: <https://ontology.okp4.space/metadata/> .
+@prefix service: <https://ontology.okp4.space/dataverse/service/> .
+@prefix topic: <https://ontology.okp4.space/thesaurus/topic/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+meta:8de7ca33-378e-445c-b087-f5ef334d2cca a dstmeta:GeneralMetadata ;
+    core:describes dataset:0ea1fc7a-dd97-4adc-a10e-169c6597bcde ;
+    core:hasCreator "Anses"^^xsd:string ;
+    core:hasDescription "Avec 3 185 aliments référencés, cette nouvelle version de la table Ciqual intègre le détail de tous les sucres individuels contenus dans les aliments. Elle propose également des données actualisées sur les principaux fruits et légumes, y compris ceux cultivés en Outre-mer. Pour être en adéquation avec les nouvelles pratiques alimentaires des français, la table inclut désormais une cinquantaine de nouveaux aliments adaptés aux régimes végétariens."@fr ;
```
Comment (ccamel) [inline example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl:16]: Please, add translation for `@en` and `@fr` as we did with cec4e7dae8361caec72e89cdcebaf86d5f3b2c47 for the other dataset instances.

File: example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+@prefix license: <https://ontology.okp4.space/thesaurus/license/> .
+@prefix mediatype: <https://ontology.okp4.space/thesaurus/media-type/> .
+@prefix meta: <https://ontology.okp4.space/metadata/> .
+@prefix service: <https://ontology.okp4.space/dataverse/service/> .
+@prefix topic: <https://ontology.okp4.space/thesaurus/topic/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+meta:8de7ca33-378e-445c-b087-f5ef334d2cca a dstmeta:GeneralMetadata ;
+    core:describes dataset:0ea1fc7a-dd97-4adc-a10e-169c6597bcde ;
+    core:hasCreator "Anses"^^xsd:string ;
```
Comment (ccamel) [inline example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl:15]: I'd remove the `xsd:string` type annotation. According to the [RDF specification](https://www.w3.org/TR/turtle/#turtle-literals), if a literal value in RDF does not have an explicit datatype IRI or a language tag, then the datatype is assumed to be `xsd:string` by default.

File: example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+@prefix dataspace: <https://ontology.okp4.space/dataverse/dataspace/> .
+@prefix dstmeta: <https://ontology.okp4.space/metadata/dataset/> .
+@prefix license: <https://ontology.okp4.space/thesaurus/license/> .
+@prefix mediatype: <https://ontology.okp4.space/thesaurus/media-type/> .
+@prefix meta: <https://ontology.okp4.space/metadata/> .
+@prefix service: <https://ontology.okp4.space/dataverse/service/> .
+@prefix topic: <https://ontology.okp4.space/thesaurus/topic/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+meta:8de7ca33-378e-445c-b087-f5ef334d2cca a dstmeta:GeneralMetadata ;
```
Comment (ccamel) [inline example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl:13]: I would add the type `owl:NamedIndividual` for all the instances (to be consistent with all other instances of the example folder):

```suggestion
meta:8de7ca33-378e-445c-b087-f5ef334d2cca a owl:NamedIndividual, dstmeta:GeneralMetadata ;
```

Comment (ccamel) [inline example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl:13]: Namespace is incorrect. Instances of metadata should be put under:

```
https://ontology.okp4.space/dataverse/dataset/metadata/
```

**Review by ccamel — CHANGES_REQUESTED** [review]
Almost good. A few notes to consider so that it is finalized and aligned with other instances already introduced. At your disposal if needed.

File: example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl (hunk 5 above)
Comment (CedrickArmel) [reply example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl:13]: All of intances of core:Metadata ?

**Comment by MdechampG** [comment]
@CedrickArmel I think you should add the last commits in this PR with the changes asked by @ccamel and with the translations.

**Comment by CedrickArmel** [comment]
I'll close this PR then open another one with all changes requested concerning the new resources to add.

@ccamel @MdechampG @fredericvilcot

**Pull request closed without merging**